src/ServiceControl.Monitoring/ServiceControl.Monitoring.csproj

```diff
@@ -214,6 +214,12 @@
   <ItemGroup>
     <Analyzer Include="..\packages\NServiceBus.7.1.4\analyzers\dotnet\cs\NServiceBus.Core.Analyzer.dll" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\DummyNetStandard20Ref\DummyNetStandard20Ref.csproj">
+      <Project>{6b406087-d062-4619-a56d-538b242f06ac}</Project>
+      <Name>DummyNetStandard20Ref</Name>
```

Comment:
Does this mean `DummyNetStandard20Ref.dll` will show up among the files?
In case the answer is "yes", I'd only suggest to replace "dummy" with something more meaningful/descriptive. In case the answer is "no", call me dummy and ignore the comment 🙂